### PR TITLE
Remove Safari code

### PR DESCRIPTION
**Pull request by jonathanKingston**
**Reviewer:** @kdzwinel @sammacbeth 

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: shared/js/content-scope/protections.js [hunk 1]
```diff
@@ -29,8 +29,7 @@ export async function loadProtections () {
         'gpc',
         'hardware',
         'referrer',
-        'screen-size',
-        'temporary-storage'
```
Comment (kdzwinel) [inline shared/js/content-scope/protections.js:33]: This is in Chrome :) We should leave it as is until Chrome completely removes it.

webkit-chrome

File: README.md [hunk 2]
```diff
@@ -15,13 +15,6 @@ DuckDuckGo Privacy Extensions is distributed under the Apache 2.0 [License](LICE
 
 [Edge](https://microsoftedge.microsoft.com/addons/detail/duckduckgo-privacy-essent/caoacbimdbbljakfhgikoodekdnlcgpk)
 
-### Safari
```
Comment (kdzwinel) [inline README.md:18]: this tells people where to find the actual Safari extension, I think we should keep it

**Review by kdzwinel — COMMENTED** [review]
I did a quick scan, found two things that we shouldn't remove, but other than that LGTM!

File: shared/js/content-scope/protections.js (hunk 1 above)
Comment (jonathanKingston) [reply shared/js/content-scope/protections.js:33]: Ah good to know!

File: shared/js/background/load.es6.js [hunk 3]
```diff
@@ -61,7 +61,7 @@ function loadExtensionFile (params) {
             xhr.setRequestHeader('If-None-Match', params.etag)
         }
     } else {
-        // set type xhr type tag. Safari internal xhr requests
+        // set type xhr type tag.
         // don't set a 200 status so we'll check this type
         xhr.type = 'internal'
```
Comment (jonathanKingston) [inline shared/js/background/load.es6.js:66]: @jdorweiler I think this else statement can be dropped actually, could you confirm please?

Comment (jonathanKingston) [reply shared/js/background/load.es6.js:66]: Actually I've checked, ignore this :)

File: Gruntfile.js [hunk 4]
```diff
@@ -132,13 +132,7 @@ module.exports = function (grunt) {
                     ['babelify'],
                     [(file) => {
```
Comment (kdzwinel) [inline Gruntfile.js:133]: have you tried dropping this whole thing? looks like a possible noop

File: unit-test/data/protections.json [hunk 5]
```diff
@@ -11,12 +11,6 @@
         ],
         "scripts": []
     },
-    "temporary-storage": {
```
Comment (kdzwinel) [inline unit-test/data/protections.json:14]: I don't think we should be removing that?

**Review by kdzwinel — COMMENTED** [review]
After testing I decided to review the code again and found some nits, sorry. Should be quick to fix and then ready to go.

BTW we also have `build/safari` in gitignore, should we remove?

File: unit-test/data/protections.json (hunk 5 above)
Comment (jonathanKingston) [reply unit-test/data/protections.json:14]: Agreed... the whole file should be removed in https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/728/ actually.

File: Gruntfile.js (hunk 4 above)
Comment (jonathanKingston) [reply Gruntfile.js]: Ah yeah, will try checking.

**Comment by kdzwinel** [comment]
Woohoo 1k LOC less 🧹 Thanks a lot!

**Pull request merged**